spring-boot/line-bot-spring-boot-handler/src/test/java/com/linecorp/bot/spring/boot/handler/support/ReplyByReturnValueConsumerTest.java

```diff
@@ -0,0 +1,169 @@
+
+        // Verify
+        verify(messagingApiClient, only())
+                .replyMessage(new ReplyMessageRequest(EVENT.replyToken(),
+                        singletonList(new TextMessage("Reply Text")),
+                        false
+                ));
+    }
+
+    @Test
+    public void acceptCompletableSingleReplyTest() throws Exception {
+        // Do
+        final CompletableFuture<TextMessage> returnValue = new CompletableFuture<>();
+        target.accept(returnValue);
+        returnValue.complete(new TextMessage("Reply Text"));
+
+        // Verify
+        verify(messagingApiClient, only())
+                .replyMessage(new ReplyMessageRequest(EVENT.replyToken(),
+                        singletonList(new TextMessage("Reply Text")),
+                        false));
+    }
+
+    @Test
+    public void acceptCompletableListReplyTest() throws Exception {
+        // Do
+        final CompletableFuture<List<TextMessage>> returnValue = new CompletableFuture<>();
+        target.accept(returnValue);
+        returnValue.complete(singletonList(new TextMessage("Reply Text")));
+
+        // Verify
+        verify(messagingApiClient, only())
+                .replyMessage(new ReplyMessageRequest(EVENT.replyToken(),
+                        singletonList(new TextMessage("Reply Text")),
+                        false));
+    }
+
+    //    @Test
+    //    public void errorInCompletableLoggingTest() throws Exception {
+    //        // Do
+    //        String systemOut = tapSystemOut(() -> {
+    //            final CompletableFuture<List<TextMessage>> returnValue = new CompletableFuture<>();
+    //            target.accept(returnValue);
+    //            returnValue.completeExceptionally(
+    //                    new GeneralLineMessagingException("EXCEPTION HAPPEN!", null, null));
+    //        });
+    //
+    //        // Verify
+    //        assertThat(systemOut)
+    //                .contains("EXCEPTION HAPPEN!");
+    //    }
+
+    //    @Test
+    //    public void errorInMessagingApiClientLoggingTest() throws Exception {
+    //        reset(MessagingApiClient);
+    //        when(MessagingApiClient.replyMessage(any()))
+    //                .thenReturn(new CompletableFuture<BotApiResponse>() {{
+    //                    completeExceptionally(new GeneralLineMessagingException("EXCEPTION HAPPEN!", null, null));
+    //                }});
+    //
+    //        // Do
+    //        String systemOut = tapSystemOut(() -> {
+    //            final CompletableFuture<List<TextMessage>> returnValue = new CompletableFuture<>();
+    //            target.accept(returnValue);
+    //            returnValue.complete(singletonList(new TextMessage("Reply Text")));
+    //        });
+    //
+    //        // Verify
+    //        assertThat(systemOut)
+    //                .contains("failed")
+    //                .contains("EXCEPTION HAPPEN!");
+    //    }
```

Comment:
Is this still necessary...?

Reply:
i'll check it

Reply:
I removed this. This test case is meaningless...